docs/pages/modules/additional-platform-support.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: Additional platform support
+description: Learn how to add support for macOS and tvOS platforms.
+---
+
+import { Step } from '~/ui/components/Step';
+
+Expo Modules API provides the first-class support for Android and iOS, however since all Apple platforms are based on the same foundation and use the same programming language,
+it is possible to target other Apple platforms in the Expo module. At the moment only **macOS** and **tvOS** platforms (also called [Out-of-Tree platforms](https://reactnative.dev/docs/out-of-tree-platforms)) are supported.
+This guide will walk you through adding support for these platforms.
+
+<Step label="1">
+
+## Use the `"apple"` platform in `expo-module.config.json`
+
+To provide seamless support for other Apple platforms, Expo SDK 50 introduced a universal `"apple"` platform to instruct the autolinking that the module may support any of the Apple platform
+and the question whether to link the module in the specific CocoaPods target is moved off to the podspec.
+If you have used `"ios"` before, you can safely replace it:
+
+```diff expo-module.config.json
+{
+-   "platforms": ["ios"],
+-   "ios": {
+-     "modules": ["MyModule"]
+-   }
++   "platforms": ["apple"],
++   "apple": {
++     "modules": ["MyModule"]
++   }
+}
+```
+
+</Step>
+<Step label="2">
+
+## Update the podspec to declare support for other platforms
+
+The module's podspec needs to be updated with a list of the supported platforms. Otherwise, CocoaPods would fail to install the pod on targets for the other platforms.
+As mentioned in the first step, this part of the spec is the source of truth for autolinking when the module is configured with a universal `"apple"` platform.
+
+```diff Module's podspec
+- s.platform       = :ios, '13.4'
++ s.platforms = {
++   :ios => '13.4',
++   :tvos => '13.4',
++   :osx => '10.15'
++ }
+```
+
+Any changes in the podspec require running `pod install` to have an effect.
+
+</Step>
+
+<Step label="3">
+
+## Set up `react-native-macos` or `react-native-tvos` in the app
+
+If you are writing a local module and your app is already set up, you can skip this step.
+Otherwise, you will need to set up your app or the example app if you are writing a standalone (non-local) module.
+
+- **For macOS**: follow the official [Install React Native for macOS](https://microsoft.github.io/react-native-windows/docs/rnm-getting-started#install-the-macos-extension) guide from `react-native-macos` documentation.
+- **For tvOS**: follow the instructions in the [react-native-tvos](https://github.com/react-native-tvos/react-native-tvos) repository.
```

Comment:
```suggestion
- **For tvOS**: follow the instructions in the [react-native-tvos](https://github.com/react-native-tvos/react-native-tvos) repository. If you are building an Expo app, you should also follow the instructions in our [Building Expo apps for TV guide](https://docs.expo.dev/guides/building-for-tv/).
```

Reply:
```suggestion
- **For tvOS**: follow the instructions in the [react-native-tvos](https://github.com/react-native-tvos/react-native-tvos) repository. If you are building an Expo app, you should also follow the instructions in our [Building Expo apps for TV guide](/guides/building-for-tv/).
```

Version of suggestion with an internal link to other doc, to avoid full re-render.